RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -126,14 +126,13 @@ def _extend_pixelLess(x):
     x.setsToMerge[0].tLists += [6]
 (trackingPhase2PU140 & vectorHits).toModify(earlyGeneralTracks, _extend_pixelLess)
 
-def _dropIter(mod, iteration):
-     mod.TrackProducers.pop(iteration)
-     mod.hasSelector.pop(iteration)
-     mod.indivShareFrac.pop(iteration)
-     mod.selectedTrackQuals.pop(iteration)
-     mod.setsToMerge[0].tLists.pop(iteration)
-
 from Configuration.ProcessModifiers.trackingLST_cff import trackingLST
-# remove initialStep from earlyGeneralTracks inputs
-(trackingPhase2PU140 & trackingLST).toModify(earlyGeneralTracks, func=lambda x:_dropIter(x,0))
-(trackingPhase2PU140 & trackingLST).toModify(earlyGeneralTracks, indivShareFrac = {0:  1})
+(trackingPhase2PU140 & trackingLST).toModify(earlyGeneralTracks,
+                         TrackProducers = ['highPtTripletStepLSTpTracks', 'highPtTripletStepLSTT5Tracks'],
+                         hasSelector = [1,0],
+                         indivShareFrac = [0.1,0.1],
```

Comment:
This is changed from `0.16` to `0.1` to be consistent with the number used in HLT. The performance doesn't change, as this is a minimal change in this case, but I thought I'd be consistent with HLT.